packages/rpc/src/methods/keys.ts

```diff
@@ -0,0 +1,36 @@
+import { requestFromNearRpc } from '../util';
+
+export type AccessKey = {
+  public_key: string;
+  permission: AccessKeyPermissions;
+};
+
+export type AccessKeyPermissions = 'FullAccess' | {
+  'FunctionCall': {
+    allowance: string;
+    receiver_id: string;
+    method_names: string[];
+  };
+}
+
+export const getAccessKeys = async (accountId: string): Promise<AccessKey[]> => {
+  const res = await requestFromNearRpc({
+    jsonrpc: '2.0',
+    id: 'dontcare',
```

Comment:
I suppose this is a meaningless field and we can send whatever we want.

Reply:
I just checked the source code and it seems they simply copy it over to the response, so I guess you could use it to order things if you have an async system running, other folks might come up with more fancy use cases ;)